tests/files/test_1.php

```diff
@@ -12,7 +12,7 @@ class TEST_1
 
    public function test($input = 4) {
       if ($input > 2) {
-         return 15;
+         return 015;
```

Comment:
test should fail

Reply:
well if we a respecting base than yes. Also if we want to ignore number 0b10 but not dec 10 would it work?

Reply:
They are 2 separate numbers 0b10 would be ignored if you had 2 in the ignore list currently. so 0x2, 02, 2 and 0b1 would all be ignored by 2.. To get this to return like that we would need to add octal, hex and binary to the cases.

Reply:
Currently this happens for context
![image](https://user-images.githubusercontent.com/1102850/53805998-a380d980-3f43-11e9-9601-379eccfc7767.png)

Reply:
You won't be able to tell the difference between 0755 and 0o755 with this